include/tvm/relay/pass.h

```diff
@@ -587,12 +654,29 @@ TVM_DLL Expr ToANormalForm(const Expr& e, const Module& mod);
  */
 TVM_DLL Expr ToGraphNormalForm(const Expr& e);
 
+inline pass::Pass ToGraphNormalFormPass() {
+  runtime::TypedPackedFunc<Function(Function, Module, pass::PassContext)> pass_func =
+    [=](Function f, Module m, pass::PassContext pc) {
+    return Downcast<Function>(ToGraphNormalForm(f));
+  };
+  return pass::CreateFunctionPass(pass_func, 1, "to_graph_normal_form", {});
+}
+
 /*! \brief Aggressive constant propagation/constant folding/inlining.
  * It will do as much computation in compile time as possible.
  * It has two benefit: remove runtime overhead, and allow more optimization (typically fusion).
  * As a side effect, code size will explode.
  */
 Expr PartialEval(const Expr& e);
+
+inline pass::Pass PartialEvalPass() {
```

Comment:
do not use inline function and put all implementation in the corresponding pass file.